src/js/views/maps/viewfinder/PredictionView.js

```diff
@@ -0,0 +1,114 @@
+'use strict';
+define(
+  ['backbone', 'text!templates/maps/viewfinder/viewfinder-prediction.html'],
+  (Backbone, Template) => {
+    // The base classname to use for this View's template elements.
+    const BASE_CLASS = 'viewfinder-prediction';
+
+    /**
+     * @class PredictionView
+     * @classdesc PredictionView shows an autocomplete suggestion
+     * for the user when they are searching for a place on a map.
+     * @classcategory Views/Maps
+     * @name PredictionView
+     * @extends Backbone.View
+     * @screenshot views/maps/viewfinder/PredictionView.png
+     * @since x.x.x
+     * @constructs PredictionView
+     */
+    const PredictionView = Backbone.View.extend({
+      /**
+       * The type of View this is
+       * @type {string}
+       */
+      type: 'PredictionView',
+
+      /**
+       * The HTML class to use for this view's outermost element.
+       * @type {string}
+       */
+      className: BASE_CLASS,
+
+      /**
+       * The HTML element to use for this view's outermost element.
+       * @type {string}
+       */
+      tagName: 'li',
+
+      /**
+       * The HTML classes to use for this view's HTML elements.
+       * @type {Object<string,string>}
+       */
+      classNames: {
+        content: `${BASE_CLASS}__content`,
+      },
+
+      /**
+      * The events this view will listen to and the associated function to call.
+      * @type {Object}
+      */
+      events: { click: 'select' },
```

Comment:
curious: is an "enter" keyup event not needed here because it's handled by the input box?

Reply:
I don't think a list item could trigger a keyup event. The input box will manage the key events though.